src/node_file.cc

```diff
@@ -491,7 +492,7 @@ int FileHandle::ReadStart() {
     BaseObjectPtr<FileHandleReadWrap> read_wrap =
         std::move(handle->current_read_);
 
-    int result = req->result;
+    ssize_t result = req->result;
```

Comment:
```suggestion
    int result = static_cast<int>(req->result);
```
Otherwise, this would always lead to `true`: https://github.com/nodejs/node/blob/683754c6f3ea8e04d34b2dad854aca95601c14a8/src/node_file.cc#L508

Reply:
the type of `req->result` is `ssize_t`, which can represent the value `-1`